src/components/TopicMapItem/TopicMapItem.tsx

```diff
@@ -41,6 +59,20 @@ export const TopicMapItem: FC<TopicMapItemProps> = ({ item, onClick }) => {
           item.topicImage?.path ? "" : styles.noImage
         }`}
```

Comment:
Add class withNote if item has dialog

```suggestion
        } ${item.dialog?.hasNote ? styles.withNote : ""}`}
```

Reply:
done